src/dataframe/dataframe.jl

```diff
@@ -609,6 +633,31 @@ function Base.insert!(df::DataFrame, col_ind::Int, item, name::Symbol)
     insert!(df, col_ind, upgrade_scalar(df, item), name)
 end
 
+"""
+Merge `DataFrame`s
```

Comment:
It would be nice to give more details about what this does, and in particular to mention `join`, because people coming from R will expect `merge` to be the same as `join`.

Reply:
fixed